### PR TITLE
MLH-577 "TC-031, TC-032, TC-035"

### PR DESCRIPTION
**Pull request by Timuriagudin**

### CODE REVIEW

File: test/regression/regression.js [hunk 1]
```diff
@@ -1 +1,15 @@
...
+
+    it('TC-035 Input 5 in Age field, then clear field', function () {
+        browser.url('');
+        const name1 = $(sel.age).setValue(data.age);
+        $(sel.age).doubleClick();
+        browser.keys("Delete");
+        const text = $(sel.explain).getProperty('innerText');
+        expect(text).toEqual(exp.required);
+    });
+});
```
Comment (vasylynash) [inline test/regression/regression.js:15]: Add an empty line at the end of the file

File: data/testData.json [hunk 2]
```diff
@@ -17,4 +18,4 @@
     "Comedy": 6
   },
   "pathToFile": "../resources/Liberty.jpg"
-}
+}
```
Comment (vasylynash) [inline data/testData.json:21]: Add an empty line at the end of the file

File: test/smoke/spinners.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import {expect} from "chai";
+
+const sel = require ('../../data/selectors.json');
+const exp = require ('../../data/expected.json');
+
+
```
Comment (vasylynash) [inline test/smoke/spinners.js:6]: Remove an empty line

File: test/smoke/spinners.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import {expect} from "chai";
+
```
Comment (vasylynash) [inline test/smoke/spinners.js:2]: Remove an empty line

File: test/smoke/spinners.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+const sel = require ('../../data/selectors.json');
+const exp = require ('../../data/expected.json');
+
+
+describe('Spinners age working', function () {
+    it('TC-032 Verify that after the first click on the spinner up the value become 1', function () {
```
Comment (vasylynash) [inline test/smoke/spinners.js:8]: Add an empty line before it

**Review by vasylynash — CHANGES_REQUESTED** [review]

File: test/regression/regression.js [hunk 6]
```diff
@@ -12,4 +12,4 @@ describe('Error "Required" appears after input field filled and cleared', functi
         const text = $(sel.explain).getProperty('innerText');
         expect(text).toEqual(exp.required);
```
Comment (vasylynash) [inline test/regression/regression.js:13]: toEqual is not from Chai.js

File: data/testData.json [hunk 7]
```diff
@@ -18,4 +18,4 @@
     "Comedy": 6
   },
   "pathToFile": "../resources/Liberty.jpg"
-}
\ No newline at end of file
+}
```
Comment (vasylynash) [inline data/testData.json]: Add an empty line at the end (maybe you can add it during "Resolve conflicts")

**Review by vasylynash — CHANGES_REQUESTED** [review]

**Review by Anastasiia-Bandurova — APPROVED** [review]

**Review by vasylynash — APPROVED** [review]

**Pull request merged**